inc/plugin.class.php

```diff
@@ -1732,7 +1753,7 @@ public static function isPluginLoaded($name) {
     */
    public static function setLoaded($id, $name) {
       global $GLPI_CACHE;
-      $plugins = $GLPI_CACHE->get('plugins');
+      $plugins = $GLPI_CACHE->get('plugins', []);
```

Comment:
This will prevent any problem if call is made prior to cache key initialization (this should not happen).